### PR TITLE
Optintopics

### PR DESCRIPTION
**Pull request by voisardf**
Adaption of the models file and other fixes for the federal models. @kalbermattenm : please review. Thanks

### CODE REVIEW

File: crdppf/models.py [hunk 1]
```diff
@@ -235,39 +233,49 @@ class CHRailwaysConstructionLimits(GeoInterface,Base):
...
+        geom = GeometryColumn(Geometry(2,srid=srid_))
+        
+    class CHAirportProjectZonesPDF(GeoInterface,Base):
+        __tablename__ = 'r103_bazl_projektierungszonen_flughafenanlagen_pdf'
+        __table_args__ = {'schema': db_config['schema'], 'autoload': True}
+        idobj = Column(Integer, primary_key=True)
+        geom = GeometryColumn(Geometry(2,srid=srid_))
+        # ch.bazl.projektierungszonen-flughafenanlagen.oereb
 
+    GeometryDDL(CHAirportProjectZonesPDF.__table__)
```
Comment (kalbermattenm) [inline crdppf/models.py]: `GeometryDDL` is only needed if you plan to insert, update or delete data in the table...

Comment (voisardf) [inline crdppf/models.py]: yep, I definitely do :) (federal feature service).

Comment (voisardf) [inline crdppf/models.py]: well, maybe in the future only for this topic...

File: crdppf/util/pdf_functions.py [hunk 2]
```diff
@@ -249,6 +249,17 @@ def get_XML(geometry, topicid, extracttime, lang, translations):
...
                                 else:
                                     geom = polygon
+            elif xtfgeom.getElementsByTagName("Punkt"):
+                point = xtfgeom.getElementsByTagName("Punkt")[0]
+                coordlist = []
+                for coords in point.childNodes:
+                    coordlist.append((
+                        float(coords.getElementsByTagName("C1")[0].firstChild.data), 
+                        float(coords.getElementsByTagName("C2")[0].firstChild.data)
+                        ))
```
Comment (kalbermattenm) [inline crdppf/util/pdf_functions.py]: Here you are appending tuples to a list ? This is what you want to do, right ?

**Comment by kalbermattenm** [comment]
Just two comments, otherwise looks good.

Merge if you are happy with it.

**Comment by voisardf** [comment]
thanks! merging

**Pull request merged**